packages/webview_flutter/webview_flutter_wkwebview/lib/src/v4/src/webkit_proxy.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../foundation/foundation.dart';
+import '../../web_kit/web_kit.dart';
+
+/// Handles constructing objects and calling static methods for the WebKit
+/// native library.
+class WebKitProxy {
+  /// Constructs a [WebKitProxy].
+  const WebKitProxy({
+    this.onCreateWebView = WKWebView.new,
```

Comment:
Why the "on" naming? These sound like observers rather than implementations. I would expect something that actually creates web views to be called `createWebView` (and the same for the others).